src/com/codersbay/PersonManagment.java

```diff
@@ -0,0 +1,44 @@
+package com.codersbay;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class PersonManagment {
+
+    public static ArrayList<Person> allPersons = new ArrayList<>();
```

Comment:
für diese Klasse sollten wir auf jedenfall jetzt noch ein paar Tests schreiben 😉

Vorschläge

* Was soll passieren wenn du auf der PersonManagment Klasse etwas entfernen möchtest wenn du noch nichts hinzugefügt hast?
* Verifizier dass die Größe von allPersons 0 ist wenn du vorher noch keine Person hinzugefügt hast
* Verifizier dass die Größe nach dem aufruf von addPerson eins größer ist also vorher
* Verifizier dass die Größe gleich bleibt wenn du remove aufrufst mit einer person die nicht in der Liste existiert